python/kafkaopmon/OpMonPublisher.py

```diff
@@ -0,0 +1,136 @@
+#!/usr/bin/env python3
+
+from opmonlib.opmon_entry_pb2 import OpMonValue, OpMonId, OpMonEntry 
+from google.protobuf.message import Message as msg
+from google.protobuf.descriptor import FieldDescriptor as fd
+from google.protobuf.timestamp_pb2 import Timestamp
+from erskafka.ERSKafkaLogHandler import ERSKafkaLogHandler
+
+import os
+import logging
+
+from datetime import datetime
+from kafka import KafkaProducer
+from typing import Optional
+
+class OpMonPublisher:
+    def __init__(
+                    self, 
+                    default_topic:str,
+                    bootstrap:str = "monkafka.cern.ch:30092", # Removed for if we don't want to use OpMon (i.e. for ssh-standalone)
+                    ers_session:str = "session_tester"
+    ) -> None:
+        ## Options from configurations
+        self.bootstrap = bootstrap
+        self.default_topic = default_topic
```

Comment:
I would like the protection against topics non starting with `monitoring.`. It was there before, not sure why it disappeared.

Reply:
In fact, it's not a protection, it's a proper addition. In the same way we do in the subscriber.
https://github.com/DUNE-DAQ/kafkaopmon/blob/582bcdbb1035239781c9593ddc3a245087445936/python/kafkaopmon/OpMonSubscriber.py#L108
`monitoring` should always be added. Same here.

Reply:
In that case what do the channels control.session.controller_name mean? There are a set of topics of this name in the dashboard. I made the change to align with those, but I can revert it

Reply:
those are topics used by drunc, which is not part of the monitoring infrastructure, hence not under my control (pun not intended). As the publisher is an object related to monitoring, all its topics have to be protected by starting with `monitoring.`.